### ISSUE TITLE
Remove the Runtime and Security Model for Web Applications spec

### ISSUE THREAD

**Issue by sideshowbarker** [dret/HTML5-overview#20]
The **Runtime and Security Model for Web Applications** spec should be removed.

Work on that spec was actually abandoned by the System Applications WG quite a long time ago. See https://lists.w3.org/Archives/Public/public-sysapps/2015Apr/0017.html where it’s noted “Approach abandoned by WG while WG was still functioning.“

That spec is just limbo/zombie state due to the fact that the System Applications WG is in fact no longer functioning as a group and has no actual active chair and is no longer actually even making decisions about things like closing up loose ends by moving their abandoned work to Notes.

Note for example the fact that [I sent a message about the Runtime and Security Model draft to the WG mailing list more than a week ago](https://lists.w3.org/Archives/Public/public-sysapps/2015Jun/0000.html) but didn’t get a single response. And note that my message was only one of two messages that have been posted to the list in the last 2 months. https://lists.w3.org/Archives/Public/public-sysapps/

So while it’s not completely inaccurate to include the Runtime and Security Model draft in this overview as being something at W3C WD status, it is misleading to do so given that the spec has in practice been abandoned and has zero chance of ever getting implemented and becoming part of the Web platform.

**Comment by dret** [comment]
i understand your point. but i am hesitant to switch things around only because there are some process difficulties within the W3C. after all, then i would be the one deciding which WDs are really truly WDs under development, and which ones got dropped on the floor. i am sure there will be cases where different people would answer this question differently, so just sticking to what is visible is certainly the easiest route (but definitely not the whole story). what about adding the ability to have some metadata for specs marking them as limbo/abandoned, and then rendering that in parentheses in the list?
(same answer applies to #21, #22, and #23)

**Comment by sideshowbarker** [comment]
> what about adding the ability to have some metadata for specs marking them as limbo/abandoned, and then rendering that in parentheses in the list?

If this overview is going to continue to list abandoned specs at all, then yeah I guess marking them each explicitly as such would be an improvement. But I think it would be much better to not list them at all in the overview—either by just dropping them completely, or by moving them out to a separate document.

> but i am hesitant to switch things around only because there are some process difficulties within the W3C.

It's not about process difficulties at the W3C. I think it's instead about the overview presenting the most useful information to readers, and not misleading and confusing and counterproductive information.

> after all, then i would be the one deciding which WDs are really truly WDs under development, and which ones got dropped on the floor. i am sure there will be cases where different people would answer this question differently, so just sticking to what is visible is certainly the easiest route (but definitely not the whole story)

I think some editorial choices were already made in putting together the current set of specs shown in the overview. First of all, obviously it doesn't include all specs published by the W3C. So some choices need to be made about which subset of W3C specs to include. One choice that was made was to include any specs at all from the SysApps WG.

I personally would never have included any of the SysApps specs to begin with, because none of them were ever specs intended to be exposed to applications on the Web to be run by browsers.

Instead at the name ”system applications” was chosen to indicate, they were specs for system-level features (as opposed to userland features)—in off-Web contexts such as ChromeOS and FirefoxOS, which don’t run under the Web security model nor under the Web runtime model. They were never specs for actual Web developers—the Web developers who I would think are intended as the primary audience for this overview.

So yeah, dropping these abandoned specs from the overview now would be an editorial decision. But no more so then the decision to include them in the first place was.

In other words, this overview is clearly already a curated list, with editorial judgements made about what to include and what not to include. That’s actually what makes the overview valuable. Otherwise readers could just look through http://www.w3.org/TR/

**Comment by dret** [comment]
now that #19 is done, if the Runtime spec becomes a NOTE, it should become much less prominent than it is now. i'll wait for a little longer and hope for the W3C to figure this one out. because otherwise i have to start implementing something that allows me to "override" status and say something along the lines of "this one is a WD but actually should be regarded as a NOTE". doable of course, but i'll wait a little longer before i start doing it ;-)

**Comment by sideshowbarker** [comment]
> i'll wait for a little longer and hope for the W3C to figure this one out.

@draggett is still the designated team contact responsible for the deliverables in that WG, so I guess he’s the best one to follow up with for info about when to expect that spec and the similar in-limbo specs in that WG will get moved to being Notes

**Comment by dret** [comment]
any idea about a timeline, @draggett? if it's close, i'll wait it out, otherwise i might have to add a little code to deal with those special cases. after all, it would be nice for the main page (now that we have that) to only list specs that are relevant.

**Comment by draggett** [comment]
> On 9 Jul 2015, at 02:32, Erik Wilde <notifications@github.com> wrote:
> 
> any idea about a timeline, @draggett <https://github.com/draggett>? if it's close, i'll wait it out, otherwise i might have to add a little code to deal with those special cases. after all, it would be nice for the main page (now that we have that) to only list specs that are relevant.
> 

I am not sure of the context for this thread, but ...

The runtime and security drafts from SysApps were discontinued.

I am due to finish a Community Group report on trust and permissions in September.   The SysApps WG is going to be formally closed soon, and I am due to republish their public WDs as WG Notes as part of the process. 

—
   Dave Raggett <dsr@w3.org <mailto:dsr@w3.org>>

**Comment by dret** [comment]
thanks @draggett, i think that's good enough for me (seeing that at some point in time, these documents will change status). the context was @sideshowbarker's concern that with these documents in WD status, people might think that they are under active development, which they aren't.

**Comment by dret** [comment]
looks like a good deal of the WDs were now turned into NOTEs. thanks! #23 reflects (some of) these updates. anything still open, @sideshowbarker? if not, please close this issue. thanks!

**Comment by sideshowbarker** [comment]
> looks like a good deal of the WDs were now turned into NOTEs. thanks! #23 reflects (some of) these updates. anything still open, @sideshowbarker? if not, please close this issue. thanks!

I guess you mean closing issue #23? Because this issue, #20, is just about the Runtime and Security Model for Web Applications spec, and that's still included in the overview (wrongly, IMHO).

**Comment by dret** [comment]
https://github.com/dret/HTML5-overview/commit/12766bcb72b9a46b8909cb5b846f7c9e19830bc8 should resolve this issue, now that the spec has been moved from WD to NOTE.

**Comment by sideshowbarker** [comment]
Yup

**Comment by sideshowbarker** [comment]
Thanks~
